app/main.py

```diff
@@ -1,3 +1,60 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance) -> Distance:
+        if not isinstance(other, Distance):
+            return Distance(
+                km=self.km + other
+            )
+        return Distance(
+            km=self.km + other.km
+        )
+
+    def __iadd__(self, other: Distance) -> Distance:
+        if not isinstance(other, Distance):
+            self.km += other
+            return self
+        else:
+            self.km += other.km
+            return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km > other
+        return self.km > other.km
+
+    def __eq__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km == other
+        return self.km == other.km
+
+    def __le__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km <= other
+        return self.km <= other.km
+
+    def __ge__(self, other: Distance) -> bool:
+        if not isinstance(other, Distance):
+            return self.km >= other
+        return self.km >= other.km
```

Comment:
you can write these two methods in one line each. Use previously created magic methods